test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -268,8 +268,8 @@ u256 EVMInstructionInterpreter::eval(
 		return m_state.timestamp;
 	case Instruction::NUMBER:
 		return m_state.blockNumber;
-	case Instruction::DIFFICULTY:
-		return m_state.difficulty;
+	case Instruction::PREVRANDAO:
+		return (m_evmVersion < langutil::EVMVersion::paris()) ? m_state.prevrandao : (u256(1) << 64) + 1;
```

Comment:
```suggestion
		return (m_evmVersion < langutil::EVMVersion::paris()) ? m_state.difficulty : m_state.prevrandao;
```
and having both in the state is probably nicer, isn't it?

Reply:
I was thinking to add it in the `0.9.0` release, but yeah. Doesn't hurt to have it already I guess.